### ISSUE TITLE
Filter incorrectly grouping elements in section

### ISSUE THREAD

**Issue by JovialMockingbird** [hievalt/ElementHider#2]
I've found an issue where it hides too much on my news site of choice. 
I've added 'voetbal^' to the filter which correctly filters the left block but it also grabs the right one which doesn't have anything to do with the left one. They are however in the same section. Is there any way to get around this? 
![image](https://user-images.githubusercontent.com/30690488/168419880-065a1d6e-8013-4ecf-bca2-3a7076d7365e.png)
![image](https://user-images.githubusercontent.com/30690488/168419980-f55122c1-208e-4612-9ee1-ff03514ce793.png)
[Archive link](https://archive.ph/VdRlT)

**Comment by hievalt** [comment]
Hi, this is caused by the maxChildren / child element ratio parameter/logic in the code. I've now added "child element ratio" option (% value) into the latest version which allows to control this behaviour. Child element ratio value defines how many % of the total website elements can be included in the element to be hidden. If element with the keyword contains bigger % of total page elements than the set ratio then it will be ignored. Low child element ratio value -> elements are ignored more likely -> more precise hiding of the elements. Tested also on the nos.nl frontpage: with child elem ratio = 1% result was same as in your image but with 0.2%-0.9% it will hide only one of the images. With 0.1% only the header text inside the image will be hidden. Version with the child element ratio option will be released in webstore soon.
